wandb/sdk/launch/runner/kubernetes_runner.py

```diff
@@ -194,7 +194,7 @@ def _watch_job(self) -> None:
         try:
             for event in self._job_watcher.stream(
                 self.batch_api.list_namespaced_job,
-                namespace="default",
+                namespace=self.namespace,
```

Comment:
https://github.com/wandb/wandb/pull/6226/files#diff-8ea6631c99cbc07d88e28024cec144305c704436041cc90266e8ff10c2017f86R214

same change :)

Reply:
Nice! I'll close this PR